kbase-extension/static/kbase/js/widgets/appWidgets2/input/dynamicDropdownInput.js

```diff
@@ -48,15 +46,22 @@ define([
             ddOptions = spec.original.dynamic_dropdown_options || {},
             dataSource = ddOptions.data_source || 'ftp_staging',
             model = {
+                displayValue: undefined,
                 value: undefined,
-            },
-            userId = runtime.userId();
+                descriptionFields: new Set(),
+            };
         let parent, container, ui;
 
         if (typeof ddOptions.query_on_empty_input === 'undefined') {
             ddOptions.query_on_empty_input = 1;
         }
 
+        // if there's a template, process it so we have expected fields
```

Comment:
I coulda sworn all of these changes made it into develop already. This is weird.

Something to do with Github's built in merging-but-not-quite-rebasing?

Reply:
Anyway, this file shouldn't be any different from the previous PR on this ticket.